taxonomies/ingredients.txt

```diff
@@ -8841,17 +8892,101 @@ wikipedia:en:https://en.wikipedia.org/wiki/Xylanase
+#                               D A I R Y     P R O D U C T S     S E C T I O N
 #
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
+en:dairy, dairy product, dairy products, dairy food, dairy foods, milk product, milk products
+af:Suiwelproduk
+als:Milcherzeugnis
+ar:منتجات الألبان
+ast:Llacteu
+be:Малочныя прадукты
+br:Boued-laezh
+bs:Mliječni proizvodi
+bxr:Сагаан эдеэн
+ca:Làctic
+ckb:سپیاتی
+cs:Mléčný výrobek
+cy:Cynnyrch llaeth
+da:Mejeriprodukter
+de:Milcherzeugnisse, Milchprodukte, Molkereiprodukte, Milcherzeugnis, Milchprodukt, Molkereiprodukt
+el:Γαλακτοκομικά προϊόντα
+en:Dairy product
+eo:Laktoprodukto
+es:productos lácteos, leche y sus derivados, leche y derivados, derivados lácteos, lácteos, lácteo
+et:Piimatoode
+eu:Esneki
+fa:فراورده لبنی
+fi:Maitotuote
+fr:produits laitiers et dérivées, laits et dérivés y compris lactose, dérivés laitiers, produits laitiers, produit laitier, produits laitiers et dérivés, lactique, lactiques, laitage
+fy:Suvel
+gl:Produtos lácteos
+he:מוצרי חלב
+hi:दुग्ध-उत्पाद
+hy:Կաթնամթերք
+id:Produk susu
+is:Mjólkurafurð
+it:Latticini
+ja:乳製品
+kab:Ifarisen n yiɣi
+kk:Сүт тағамдары
+kn:ಕ್ಷೀರೋತ್ಪನ್ನ
+ko:유제품
+lt:Pieno produktas
+lv:Piena produkti
+mn:Сүүн бүтээгдэхүүн
+ms:Produk tenusu
+nl:Zuivel
+oc:Produch lachièr
+pl:Przetwory mleczne
+pt:Lacticínios
+ro:Produse lactate
+ru:Молочные продукты
+sco:Dairy product
+si:ගව කිරි ආශ්‍රිත නිෂ්පාදන
+sl:Mlečni izdelek
+sr:Млечни производи
+sv:Mejeriprodukt
+ta:பால் பொருள்
+th:ผลิตภัณฑ์นม
+tr:Mandıra ürünü
+uk:Молочні продукти
+uz:Sut mahsulotlari
+yi:מילכיגס
+zh:奶制品
+wikipedia:en:https://en.wikipedia.org/wiki/Dairy_product
+wikidata:en:Q185217
+vegan:en:no
+vegetarian:en:maybe
+
+<en:dairy
+en:fermented milk products, cultured dairy foods, cultured dairy products, cultured milk products
+be:Кісламалочныя прадукты
+ca:Llet fermentada
+cs:Kysaný mléčný výrobek
+de:Sauermilchprodukte, Sauermilchprodukt, Sauermilcherzeugnis, Sauermilcherzeugnisse
+en:Fermented milk products
+es:Producto fermentado de la leche
+hy:Կաթնաթթվային մթերք
+ko:발효유
+lv:Skābpiena produkti
+ru:Кисломолочные продукты
+uk:Кисломолочний продукт
+wikipedia:en:https://en.wikipedia.org/wiki/Fermented_milk_products
+wikidata:en:Q3506176
+vegan:en:no
+vegetarian:en:maybe
+
+<en:dairy
+en:powdered milk products, dried milk products, product on base on milk powder
+de:Trockenmilcherzeugnis
+
+
+################################ BUTTER related stuff ##########################
+
 # description:en:BUTTER is a dairy product with high butterfat content which is solid when chilled and at room temperature in some regions, and liquid when warmed. It is made by churning fresh or fermented cream or milk to separate the butterfat from the buttermilk. It is generally used as a spread on plain or toasted bread products and a condiment on cooked vegetables, as well as in cooking, such as baking, sauce making, and pan frying. 
```

Comment:
Ingredients and not products implies that all entries can be found on ingredients  lists. Is this the case for `en:fermented milk products`? If not, why introduce this ingredient? We do not want to redo the categories. The taxonomy tree should stay as flat as possible.

Reply:
At least in German it exists as ingredient (not very abundant), see https://world.openfoodfacts.org/ingredient/de:Sauermilcherzeugnis

Reply:
@aleene I'm fine also with reverting
```diff
-<en:fermented milk products
+<en:dairy
en:yogurt / en:kefir / en:soured milk / en:buttermilk
```
Do you prefer that?

Reply:
For the moment, yes

Reply:
Okay, done.